supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesConfiguration.scala

```diff
@@ -13,8 +17,35 @@ class TesConfiguration(val configurationDescriptor: BackendConfigurationDescript
       .backendConfig
       .as[Option[Boolean]](TesConfiguration.useBackendParametersKey)
       .getOrElse(false)
+
+  val pollBackoff =
+    configurationDescriptor
+      .backendConfig
+      .as[Option[Config]]("poll-backoff")
+      .map(SimpleExponentialBackoff(_))
+      .getOrElse(TesConfiguration.defaultPollBackoff)
+
+  val executeOrRecoverBackoff =
+    configurationDescriptor
+      .backendConfig
+      .as[Option[Config]]("execute-or-recover-backoff")
+      .map(SimpleExponentialBackoff(_))
+      .getOrElse(TesConfiguration.defaultExecOrRecoverBackoff)
+
 }
 
 object TesConfiguration {
   final val useBackendParametersKey = "use_tes_11_preview_backend_parameters"
+
+  final val defaultPollBackoff = SimpleExponentialBackoff(
+    initialInterval = 10 seconds,
```

Comment:
This `initialInterval` default value is different than what we were using before. I don't see any issue with changing from 1 -> 10seconds, but just confirming it's intentional.

Reply:
Yup, that is intentional! For nontrivial workflows polling every 1s seems silly.